docs/general/ledger.md

```diff
@@ -1,53 +1,61 @@
-hierarchical deterministic wallets (HD wallets), that is:
+The {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} [Ledger](https://www.ledger.com/)
+application is compatible with the Ledger Nano S and the Ledger Nano X devices. Ledger devices are
+hardware wallets that keep your secret key secured on a physical device that does not expose it to
+your computer or the internet. The private keys will not be leaked even if you connect your nano via
+USB to your computer. Ledger devices are also hierarchical deterministic wallets (HD wallets), that
+is:
 
 - _Deterministic_ means that only one seed phrase generates all the accounts for different
   blockchain networks.
 - _Hierarchical_ means that the accounts are generated in a tree-like structure for different
   purposes.
 
-Ledger devices can be equipped with applications that are blockchain-specific. Such applications are
-usually developed by third parties, and they enable the user to transact securely on the blockchain
-network. The Polkadot Ledger application allows you to manage Polkadot's native token, DOT. It
-supports most of the transaction types of the network, including batch transactions from the Utility
-pallet.
+Ledger devices can be equipped with applications that are blockchain-specific. Third parties usually
+develop such applications, enabling the user to transact securely on the blockchain network. The
+Polkadot Ledger application allows you to manage
+{{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} native token,
+{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}. It supports most of the transaction types of
+the network, including batch transactions from the Utility pallet.
 
 :::note Ledger apps may not support all the transactions
 
-Check [Ledger Polkadot App](https://github.com/Zondax/ledger-polkadot) specification for the list of
-transactions supported. Some transactions are supported only on a specific version of the app, and
-others are not supported at all by any version. For instance, the staking `rebag` extrinsic is
-supported on the XL version but not on the light version. Support for the XL version is limited to
-single calls (not batch calls), meaning you can rebag one account at a time. Also, joining a
-[nomination pool](../learn/learn-nomination-pools.md) is only possible with the XL version.
+Check the
+{{ polkadot: [Ledger Polkadot App](https://github.com/Zondax/ledger-polkadot) :polkadot }}{{ kusama: [Ledger Kusama App](https://github.com/Zondax/ledger-kusama) :kusama }}
+specification for the list of transactions supported. Some transactions are supported only on a
+specific app version, and others are not supported by any version. For instance, the staking `rebag`
+extrinsic is supported on the XL version but not on the light version. Support for the XL version is
+limited to single calls (not batch calls), meaning you can rebag one account at a time. Also,
```

Comment:
Batch calls are supported by all versions of the apps https://github.com/Zondax/ledger-polkadot#utility

Reply:
yep, content removed